Services/Component/classes/class.ilPlugin.php

```diff
@@ -857,7 +857,13 @@ protected function beforeUninstall()
 				$ilDB->manipulate("DELETE FROM lng_modules".
 					" WHERE module = ".$ilDB->quote($prefix, "text"));
 			}
-			
+
+			// clear the event listening
+			include_once("Services/Component/classes/class.ilPluginReader.php");
```

Comment:
The include can be ommitted, just run a `composer dumpautoload` in libs/composer and the class will be part of the classmap and will be found in the autoloader